src/dotnet/APIView/ClientSPA/src/app/_components/conversations/conversations.component.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ConversationsComponent } from './conversations.component';
+import { SharedAppModule } from 'src/app/_modules/shared/shared-app.module';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ReviewPageModule } from 'src/app/_modules/review-page/review-page.module';
+import { APIRevision } from 'src/app/_models/revision';
+import { CommentItemModel } from 'src/app/_models/commentItemModel';
+import { ActivatedRoute, convertToParamMap } from '@angular/router';
+import { of } from 'rxjs';
+
+describe('ConversationComponent', () => {
+  let component: ConversationsComponent;
+  let fixture: ComponentFixture<ConversationsComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [ConversationsComponent],
+      imports: [
+        HttpClientTestingModule,
+        ReviewPageModule,
+        SharedAppModule
+      ],
+      providers: [
+        {
+          provide: ActivatedRoute,
+          useValue: {
+            snapshot: {
+              paramMap: convertToParamMap({ reviewId: 'test' }),
+            },
+            queryParams: of(convertToParamMap({ activeApiRevisionId: 'test', diffApiRevisionId: 'test' }))
+          }
+        }
+      ]
+    });
+    fixture = TestBed.createComponent(ConversationsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  describe('createCommentThreads', () => {
```

Comment:
might be worth adding more tests to the other functionalities in the `ConversationsComponent` class like `numberOfActiveThreads`, `upVote`